.github/workflows/prc.yml

```
@@ -64,12 +83,23 @@ jobs:
         with:
           go-version: 1.21
       - uses: actions/checkout@v3
+      - name: Configure Redis module workspace
+        run: |
+          go work init . ./taskmanager/redis
+          go work edit -replace=trpc.group/trpc-go/trpc-a2a-go/v2@v2.0.0-alpha.3=.
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
```

actionlint (1.7.12)

[error] the runner of "golangci/golangci-lint-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)